### ISSUE TITLE
County EOY report out

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/health#453]

**Comment by lippytak** [comment]
- County-specific stats
- lovebombs
- feedback survey...live data? few examples?
- product roadmap or features we're going to work on next (interview rescheduling, automatic outcome data collection, sar-7)

**Comment by lippytak** [comment]
Maybe Kim and call and give some love to county partners :)

**Comment by lippytak** [comment]
Goal is to keep folks engaged + supportive so they want to work with us on growth+features+biz model next year.

Distro list:
- sf hsa leo/tiana/trent/noelle
- sfmfb outreach team
- ccc vakil + FB
- cdss will/pete/kim/jeff (w a few copy tweaks)
- solano angela/karren
- placer linda/logan
- marin ronna/alice
- sd rick
- fns andy risenberg
- other - stacey dean
- internal

**Draft**
___
Linda, Logan,

As 2015 comes to a close I wanted to share a final update and a glimpse into what's next. See below.

Thank you both for such a wonderful year and see you in 2016!

Warmly,
Jake

================================
We [made](http://citizenonboard.com/snap/ca/) [many](http://ebtnearme.org/) [things](http://www.codeforamerica.org/products/balance/) in 2015, but a few stand out:
- Built a simple online application for CalFresh (GetCalFresh.org) that takes ~10 minutes to complete and works on mobile phones. It speaks to you like a human and it Just Works.
- Started active pilot partnerships with 6 California counties and successfully processed over 1,500 CalFresh applications. In total we've helped Californians access about $1M in CalFresh benefits so far.
- Identified and measured key barriers in the CalFresh enrollment process to enable real data-driven decision making: [FUNNEL CHART]
- Here's what some of your clients are saying:
    [QUOTE QUOTE QUOTE]

In 2016 we're shooting to build the best SNAP application in the country (by far!), help make a dent in CalFresh participation, and work out a sustainable nonprofit business model. In the process, I hope we can set a new standard for how digital social services should work, and show how nonprofits and government can partner to deliver truly excellent services.

Onwards!

[maybe the summit big team/community img?]

**Comment by alanjosephwilliams** [comment]
Small suggestions:
- "With help of partners like you, we made many things"
- Include an explicit thank you. "We couldn't have done it without your leadership and the collaboration of your staff"

**Comment by alanjosephwilliams** [comment]
Kim call off the table?

**Comment by lippytak** [comment]
@daguar plz take a quick look!

**Comment by daguar** [comment]
Looks good.

- Maybe mention what % of clients are from phones, who might not have applied otherwise (or shown up in lobby)
- Maybe think more on the framing around non-profit business model -- what's the best language and goal for saying this?
- What else will get them super stoked? I think this could use one more element

**Comment by lippytak** [comment]
Thanks to endless support and guidance from a growing bullpen of partners, we were able to [make](http://citizenonboard.com/snap/ca/) [many](http://ebtnearme.org/) [things](http://www.codeforamerica.org/products/balance/) in 2015. We built a simple online application for CalFresh that takes 10 minutes and works on mobile phones. We started active pilot partnerships with 6 California counties and successfully processed over 1,500 CalFresh applications. And in the process we measured key barriers in the CalFresh enrollment process to enable data-driven and client-centered decisions:
![calfresh_funnel](https://cloud.githubusercontent.com/assets/2533112/12202607/9fc1cc32-b5df-11e5-8581-b58988c66799.png)

We learned that interviews are hard and that we deny 10-20% of CalFresh applicants before they even talk to an eligibility worker. We learned that lots of people (about 1/3) will apply for CalFresh on their phone if we give them the chance, and that the vast majority (around 90%) want to know about their application via email or text message. Most importantly, we learned from your clients that #SNAPWorks and is getting better every day: "The positive attitude of the program now is such a nice thing; when I was a child it was not upbeat at all. Thanks again!"

In 2016 we're aiming to build the best SNAP application experience in the country (by far!), help make a real dent in CalFresh participation, and show that SNAP can do more than just work. SNAP can be so good it's *surprising*: "It was incredibly easy, quick, and friendly. I expected something intimidating or burdensome, but was pleasantly surprised." SNAP can be *awesome*: "The online application was completely awesome. It took literally six or seven minutes. It was fast, simple, easy." In 2016 I want to show that getting SNAP can be easier than getting groceries.

I'm truly humbled to play a small part in making SNAP more amazing in 2015 and I can't wait to see all we can do together in the coming year.

Onwards!
